### PR TITLE
Feature: Add switch component

### PR DESCRIPTION
**Pull request by NduatiK**

### CODE REVIEW

**Comment by NduatiK** [comment]
[Add a "Switch" to Elemental](https://trello.com/c/HGc6UR2T/23-add-a-switch-to-elemental)

**Review by dhruvio — CHANGES_REQUESTED** [review]
Looks good, could you please switch from `verticalPadding` to the `spacerMultiples` approach described in https://github.com/realfolk/elm-elemental/pull/4#issuecomment-1150290361 ? All spacers should be configurable via `spacerMultiples`.

File: src/Elemental/View/Form/Field/Switch.elm [hunk 1]
```diff
@@ -21,7 +21,9 @@ type alias Options msg =
     , text : String
     , disabled : Bool
     , size : Size
-    , verticalPadding : Size -> Float
+    , spacerMultiples :
+        { y : Size -> Float
+        }
```
Comment (dhruvio) [inline src/Elemental/View/Form/Field/Switch.elm:26]: This should be a part of the `Theme`

**Review by dhruvio — CHANGES_REQUESTED** [review]
Added a comment requesting a change. Also, the spacer multiple between the handle and the text should be configurable.

File: src/Elemental/View/Form/Field/Switch.elm (hunk 1 above)
Comment (NduatiK) [reply src/Elemental/View/Form/Field/Switch.elm:26]: Any preference on the name of that spacer?

Comment (dhruvio) [reply src/Elemental/View/Form/Field/Switch.elm:26]: Do you mean the name of the spacer in between the handle and text? I don't have a preference for that :)

`spacerMultiples` however, can be included as a property inside the `Theme` record directly (see https://github.com/realfolk/elm-elemental/blob/master/src/Elemental/View/Form/Field/Select.elm#L62)

**Review by dhruvio — APPROVED** [review]
LGTM 👍🏽

**Pull request merged**